modules/masterbar/masterbar.php

```diff
@@ -667,37 +667,73 @@ public function add_my_sites_submenu( $wp_admin_bar ) {
 			),
 		) );
 
-		// Portfolio
-		$portfolios_title = $this->create_menu_item_pair(
-			array(
-				'url'   => 'https://wordpress.com/types/jetpack-portfolio/' . esc_attr( $this->primary_site_slug ),
-				'id'    => 'wp-admin-bar-edit-portfolio',
-				'label' => __( 'Portfolio', 'jetpack' ),
-			),
-			array(
-				'url'   => 'https://wordpress.com/edit/jetpack-portfolio/' . esc_attr( $this->primary_site_slug ),
-				'id'    => 'wp-admin-bar-new-portfolio',
-				'label' => _x( 'Add', 'admin bar menu new item label', 'jetpack' ),
-			)
-		);
-
-		if ( ! current_user_can( 'edit_pages' ) ) {
-			$portfolios_title = $this->create_menu_item_anchor(
-				'ab-item ab-primary mb-icon',
-				'https://wordpress.com/types/jetpack-portfolio/' . esc_attr( $this->primary_site_slug ),
-				__( 'Portfolio', 'jetpack' ),
-				'wp-admin-bar-edit-portfolio'
+		// Testimonials
+		if ( Jetpack::is_module_active( 'custom-content-types' ) && get_option( 'jetpack_testimonial' ) ) {
+			$testimonials_title = $this->create_menu_item_pair(
+				array(
+					'url'   => 'https://wordpress.com/types/jetpack-testimonial/' . esc_attr( $this->primary_site_slug ),
+					'id'    => 'wp-admin-bar-edit-testimonial',
+					'label' => __( 'Testimonials', 'jetpack' ),
+				),
+				array(
+					'url'   => 'https://wordpress.com/edit/jetpack-testimonial/' . esc_attr( $this->primary_site_slug ),
+					'id'    => 'wp-admin-bar-new-testimonial',
+					'label' => _x( 'Add', 'admin bar menu new item label', 'jetpack' ),
```

Comment:
This label is pretty confusing. "Button label for adding a new item via the toolbar menu" is a bit easier to understand I think.